### PR TITLE
Email

### PR DESCRIPTION
**Pull request by RRR777**

### CODE REVIEW

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -9,6 +9,7 @@ class HomeController extends Controller
 {
     /**
      * @Route("/", name="home")
+     * @throws
```
Comment (chyzas) [inline src/Controller/HomeController.php]: kam sitas?

File: src/Controller/ReservationController.php [hunk 2]
```diff
@@ -121,4 +121,26 @@ public function index()
...
         return $this->render('reservation/myReservations.html.twig');
     }
+
+    /**
+     * @Route("/reservation/confirm", name="confirm_reservation")
+     * @IsGranted("ROLE_USER")
+     */
+    public function confirm(\Swift_Mailer $mailer)
+    {
+        $message = (new \Swift_Message('Registration Confirmation'))
```
Comment (chyzas) [inline src/Controller/ReservationController.php]: vel angliakalbystes

File: src/Controller/ReservationController.php [hunk 3]
```diff
@@ -121,4 +121,26 @@ public function index()
...
+            ->setFrom('send@example.com')
```
Comment (chyzas) [inline src/Controller/ReservationController.php]: set from bigfishes@gmail.com ar koks ten tas emailas

File: src/Controller/ReservationController.php [hunk 4]
```diff
@@ -121,4 +121,26 @@ public function index()
...
+            ->setTo('recipient@example.com')
```
Comment (chyzas) [inline src/Controller/ReservationController.php]: set to user email

File: src/Controller/HomeController.php (hunk 1 above)
Comment (RRR777) [reply src/Controller/HomeController.php]: tada nebemeta warning dėl \DataTime :)

**Pull request merged**